charts/brig/values.yaml

```
@@ -63,6 +63,11 @@ config:
     setDefaultLocale: en
     setMaxTeamSize: 500
     setMaxConvSize: 500
+    # Who can see the emails of other users
+    # visible_to_self: only the self user can
+    # visible_if_on_same_team: users in the same team can view each others' emails
+    # visible_if_on_team: any team user can have its email viewed
+    # Ref.: https://github.com/wireapp/wire-server/blob/0126651a25aabc0c5589edc2b1988bb06550a03a/services/brig/src/Brig/Options.hs#L304-L306
```

```suggestion
    # For meaning, see https://github.com/wireapp/wire-server/blob/0126651a25aabc0c5589edc2b1988bb06550a03a/services/brig/src/Brig/Options.hs#L304-L306
```
Referencing the wire-server code and therefore point to the meaning of the values totally make sense, but adding them here too is redundant and should be avoided IMHO. In case of change, nobody might be aware of the duplication here.

---

Hmmm, to avoid duplication maybe we can just point to https://github.com/wireapp/wire-server/blob/develop/docs/reference/config-options.md and consistently describe the different options there? (And keep just the defaults here)

---

pointing directly to code would be my personal preference, because usually changes appear there first (otherwise git log ftw), and sometimes that's the only place, but sure. In this case, your suggestion has the same result.